docs/workitems.md

```diff
@@ -1,109 +1,127 @@
-# Work Items
```

Comment:
Was this change intentional? It seems like you replaced the entire workitems doc with a copy of the roadmaps doc.

Reply:
Sorry!  When I started reading I hadn't forked and then I made an error in copying the file changes.  I've fixed the file now.  Thank you!